confirmation-template.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    /*
+    Template Name: Confirmation
+	*/
+
+	get_header();
+	the_post();
+
+	$message = "";
+
+	if( isset ( $_GET['audience'] ) && have_rows( 'confirmations', $post ) ) :
+		$confirmations = get_field( 'confirmations', $post );
+		$message_key = array_search( $_GET['audience'], array_column( $confirmations, 'confirmation_parameter' ) );
+		$message = ( $message_key === FALSE || $message_key === "" ) ? "" : $confirmations[$message_key]['confirmation_message'];
+	endif;
+?>
+
+<div class="container">
+	<div class="row my-5">
```

Comment:
What do you think about updating this margin class to:
```suggestion
	<div class="row my-4 my-lg-5">
```
To shorten the mobile spacing a bit.